cmake/config/coverage.cmake

```diff
@@ -0,0 +1,43 @@
+#### -DCMAKE_BUILD_TYPE=Coverage
+
+# https://github.com/bilke/cmake-modules/blob/master/CodeCoverage.cmake#L90
+set(FLAGS
+    -O1              # first level of optimizations
+    --coverage
+
```

Comment:
Empty string, is it intended? Same with the following files

Reply:
Yes, it is a separator